src/debug/di/breakpoint.cpp

```diff
@@ -15,7 +15,7 @@
 
 CordbBreakpoint::CordbBreakpoint(CordbProcess * pProcess, CordbBreakpointType bpType)
   : CordbBase(pProcess, 0, enumCordbBreakpoint), 
-  m_active(false), m_type(bpType)
+  m_active(false), m_type(bpType), m_pAppDomain()
```

Comment:
You'll need to put the `m_pAppDomain` initialization before the `m_type` to fix the CI issue.
Also, could you please use `m_pAppDomain(NULL)` form instead to make it more readable?